_posts/news/2020-05-13-announcing-scalajs-0.6.33.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.33
+category: news
+tags: [releases]
+permalink: /news/2020/05/13/announcing-scalajs-0.6.33/
+---
+
+
+We are pleased to announce the release of Scala.js 0.6.33!
+
+This is mostly a bugfix release, including a fix for a bad interaction with React.js' development mode and the `jsdom` JS env.
```

Comment:
0.6.33 will start to warn, if one overrides `equals` or `hashCode` in a `js.Any` (scala-js/scala-js#3969). Should we talk about it here?